### ISSUE TITLE
Formaat niet-event attributen 

### ISSUE THREAD

**Issue by adgerrits** [VNG-Realisatie/notificatieservices#114]
Binnen de CE-community zijn mensen die vinden dat attributen die niet event-gebonden zijn herkenbaar moeten zijn. Te overwegen bij attributen zoals subscriberreference en subscription.

**Comment by Rateotg** [comment]
Heeft men ook een suggestie gedaan op welke manier deze herkenbaar gemaakt kunnen/moeten worden?

**Comment by adgerrits** [comment]
Nee. Was ihkv de `data_base64` discussie waarbij als alternatief 'encoding' werd voorgesteld. Reactie was dat een vd actiefste mensen dan liever 'data_encoding' zou zien om expliciet te maken dat het attribuut alleen betrekking heeft opmhet attribuut 'data'.
